stylecop.ruleset

```diff
@@ -116,7 +116,7 @@
     <!-- Maintainability rules -->
     <Rule Id="SA1400" Action="Error" /> <!-- Access modifier must be declared -->
     <Rule Id="SA1401" Action="Error" /> <!-- Fields must be private -->
-    <Rule Id="SA1402" Action="Error" /> <!-- File may only contain a single class -->
+    <Rule Id="SA1402" Action="None" /> <!-- File may only contain a single class -->
```

Comment:
Well, as suggested in Color3.cs and Color4.cs, If you move the types out there, correct that into an error here again.